benchmark/utils/DriverUtils.swift

```diff
@@ -268,7 +298,11 @@ func runBench(_ name: String, _ fn: (Int) -> Void, _ c: TestConfig) -> BenchResu
     var elapsed_time : UInt64 = 0
     if c.fixedNumIters == 0 {
       elapsed_time = sampler.run(name, fn: fn, num_iters: 1)
-      scale = UInt(time_per_sample / elapsed_time)
+      if elapsed_time > 0 {
```

Comment:
@gottesmm ```elapsed_time == 0``` happened with empty bodies when I used low-res timer ```clock(3)```.  It is rare but it may still happen, and I don't want to give up all results when it happens to a test. Do you think ```stderr``` can be used for this purpose (i.e. it doesn't jeopardize results)?